test/e2e/operator_groups_e2e_test.go

```diff
@@ -340,27 +340,31 @@ var _ = Describe("Operator Group", func() {
 		})
 
 		// validate provided API clusterroles for the operatorgroup
-		adminRole, err := c.KubernetesInterface().RbacV1().ClusterRoles().Get(context.TODO(), operatorGroup.Name+"-admin", metav1.GetOptions{})
-		require.NoError(GinkgoT(), err)
-		adminPolicyRules := []rbacv1.PolicyRule{
-			{Verbs: []string{"*"}, APIGroups: []string{mainCRD.Spec.Group}, Resources: []string{mainCRDPlural}},
-		}
-		require.Equal(GinkgoT(), adminPolicyRules, adminRole.Rules)
-
-		editRole, err := c.KubernetesInterface().RbacV1().ClusterRoles().Get(context.TODO(), operatorGroup.Name+"-edit", metav1.GetOptions{})
+		existingClusterRoleList, err := c.KubernetesInterface().RbacV1().ClusterRoles().List(context.TODO(), metav1.ListOptions{
+			LabelSelector: labels.SelectorFromSet(ownerutil.OwnerLabel(&operatorGroup, "OperatorGroup")).String(),
+		})
 		require.NoError(GinkgoT(), err)
-		editPolicyRules := []rbacv1.PolicyRule{
-			{Verbs: []string{"create", "update", "patch", "delete"}, APIGroups: []string{mainCRD.Spec.Group}, Resources: []string{mainCRDPlural}},
-		}
-		require.Equal(GinkgoT(), editPolicyRules, editRole.Rules)
+		require.Len(GinkgoT(), existingClusterRoleList.Items, 3)
 
-		viewRole, err := c.KubernetesInterface().RbacV1().ClusterRoles().Get(context.TODO(), operatorGroup.Name+"-view", metav1.GetOptions{})
-		require.NoError(GinkgoT(), err)
-		viewPolicyRules := []rbacv1.PolicyRule{
-			{Verbs: []string{"get"}, APIGroups: []string{"apiextensions.k8s.io"}, Resources: []string{"customresourcedefinitions"}, ResourceNames: []string{mainCRD.Name}},
-			{Verbs: []string{"get", "list", "watch"}, APIGroups: []string{mainCRD.Spec.Group}, Resources: []string{mainCRDPlural}},
+		for _, role := range existingClusterRoleList.Items {
+			if strings.HasSuffix(role.Name, "admin") {
+				adminPolicyRules := []rbacv1.PolicyRule{
+					{Verbs: []string{"*"}, APIGroups: []string{mainCRD.Spec.Group}, Resources: []string{mainCRDPlural}},
+				}
+				require.Equal(GinkgoT(), adminPolicyRules, role.Rules)
```

Comment:
if possible, do something like

```go
if diff := cmp.Diff(adminPolicyRules, role.Rules); diff != "" {
    fmt.Println(diff) // Ginkgo mangles this output horribly ...
    GinkgoT().Fail("invalid rules on admin role")
}
```

The next person to break this will thank you

Reply:
if `assert` were imported, this would be a smidgen easier...